commons/src/test/java/org/eclipse/kapua/commons/about/AboutScannerTest.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua.commons.about;
+
+import org.eclipse.kapua.qa.markers.junit.JUnitTests;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.reflections.util.ClasspathHelper;
+
+import java.lang.reflect.Field;
+import java.util.LinkedList;
+import java.util.List;
+
+@Category(JUnitTests.class)
+public class AboutScannerTest extends Assert {
+
+    @Test
+    public void scanTest() {
+        final AboutScanner scanner = AboutScanner.scan();
+        assertNotNull("Null not expected", scanner);
```

Comment:
For example in this case this makes sense.
There is no `new` and `AboutScanner.scan()` can return `null` even if is not designed to, if someone changes the implementation of `scan()`